manager/allocator/network.go

```diff
@@ -111,6 +111,21 @@ func (a *Allocator) doNetworkInit(ctx context.Context) error {
 		if err := a.allocateNetwork(ctx, nc, ingressNetwork); err != nil {
 			log.G(ctx).Errorf("failed allocating ingress network during init: %v", err)
 		}
+
+		// Update store after allocation
+		if err := a.store.Update(func(tx store.Tx) error {
+			if err := store.UpdateNetwork(tx, ingressNetwork); err != nil {
+				return err
+			}
+
+			return nil
+		}); err != nil {
+			return fmt.Errorf("failed to create ingress network: %v", err)
+		}
+
+		a.store.View(func(tx store.ReadTx) {
+			ingressNetwork = store.GetNetwork(tx, ingressNetwork.ID)
```

Comment:
Why doing a Get right after an Update? The `ingressNetwork` on hand should already be the latest version